versioning/lockfiles/configurations.rst

```diff
@@ -0,0 +1,191 @@
+.. _versioning_lockfiles_configurations:
+
+Multiple configurations
+=======================
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+In the previous section we managed just 1 configuration, for the default profile. In many applications,
+packages needs to be built with several different configurations, typically managed by different profile
+files.
+
+.. note::
+
+    This section continues with the previous example with the :ref:`versioning_lockfiles_introduction`.
+    The code used in this section, including a *build.py* script to reproduce it, is in the
+    examples repository: https://github.com/conan-io/examples. You can go step by step
+    reproducing this example while reading the below documentation.
+
+    .. code:: bash
+
+        $ git clone https://github.com/conan-io/examples.git
+        $ cd features/lockfiles/intro
+        # $ python build.py only to run the full example, but better go step by step
+
+
+Lets start in the *features/lockfiles/intro* of the examples repository, remove the previous packages,
+and create both release and debug ``pkga`` packages:
+
+.. code-block:: bash
+
+    $ conan remove "*" -f
```

Comment:
Maybe we can do `conan remove "pkg*" -f` so that the user that is following the tutorial does not delete the entire cache accidentally?

Reply:
Done